packages/client/wallets/aa/src/SmartWalletSDK.ts

```diff
@@ -27,15 +27,15 @@ export class CrossmintAASDK extends LoggerWrapper {
         this.eaoWalletService = new EOAWalletService(this.crossmintService);
     }
 
-    static init(params: CrossmintAASDKInitParams): CrossmintAASDK {
-        return new CrossmintAASDK(params);
+    static init(params: SmartWalletSDKInitParams): SmartWalletSDK {
+        return new SmartWalletSDK(params);
     }
 
-    async getOrCreateWallet(
+    async getOrCreate(
```

Comment:
Im not sure this change makes sense to me

SmartWalletSDK.get() seems like it would return an SDK, not a wallet?

Reply:
reverted!